src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/MaskedTextProvider.cs

```diff
@@ -19,6 +19,7 @@ namespace System.ComponentModel
     /// to the test string itself, since mask literals cannot be modified (i.e: replacing on a literal position
     /// will actually replace on the nearest edit position forward).
     /// </summary>
+    [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
```

Comment:
I know it's ... not polite... but I have to say that I'm supper happy with the linker feature for this. One-line fix for a gnarly problem.

Nit: I might add a comment describing where/how is this annotation needed - that's the one thing linker doesn't make obvious in this case.